catalog/app/containers/Bucket/requests.js

```diff
@@ -379,6 +379,7 @@ export const search = async ({ es, query }) => {
     const result = await es({
       query,
       fields: ['content', 'comment', 'key_text', 'meta_text'],
+      type: 'cross_fields',
```

Comment:
@akarve `es` function doesn't support `type` argument currently. i'll tweak it to do that in a sec